packages/stk/stk_balance/test_utils/BalanceTestUtilities.cpp

```diff
@@ -1,6 +1,6 @@
 #include "BalanceTestUtilities.hpp"
-#include <gtest/gtest.h>
 #include <iomanip>
+#include <unistd.h>
```

Comment:
That's a POSIX header file -- do you actually need POSIX, non-C++ functions?

Reply:
Unfortunately this file uses the 'unlink' function.